@xen-orchestra/web/src/components/vm/dashboard/VmDashboardQuickInfo.vue

```diff
@@ -11,11 +11,29 @@
       </VtsQuickInfoRow>
       <VtsQuickInfoRow :label="t('ip-address')" :value="vm.mainIpAddress" />
       <VtsQuickInfoRow :label="t('created-on')" :value="installDateFormatted" />
+      <VtsQuickInfoRow :label="t('created-by')" :value="user?.email ?? t('unknown')" />
       <VtsQuickInfoRow :label="t('started')" :value="relativeStartTime" />
+    </VtsQuickInfoColumn>
+    <VtsQuickInfoColumn>
+      <VtsQuickInfoRow :label="t('uuid')" :value="vm.id" />
+      <VtsQuickInfoRow :label="t('pool')">
+        <template #value>
+          <span v-if="pool" class="host-name">
```

Comment:
```suggestion
          <span v-if="pool" class="pool-name">
```
Or rename it to use the same class for Pool and Host